src/CfgEditor/CfgEditorPanel.ts

```diff
@@ -83,7 +85,7 @@ export class CfgEditorPanel implements vscode.CustomTextEditorProvider {
     };
     vscode.commands.executeCommand('setContext', CfgEditorPanel.viewType, true);
 
-    webviewPanel.webview.html = this._getHtmlForWebview(webviewPanel.webview);
+    webviewPanel.webview.html = (await this._getHtmlForWebview(webviewPanel.webview)).toString();
```

Comment:
I think `toString` in this line can be removed. 

As you already know, but for others information, `Buffer.from` method returns `array<uint8>` type.

But, below L147 is already calling `toString` method.
```ts
Buffer.from(await vscode.workspace.fs.readFile(htmlPath)).toString();
```

So, you've got two option.
1. Remove `toString` method in this line.
2. Make `_getHtmlForWebview` return `array<uint8>` type.

But seems first one looks better because it would be easier to modify html contents before return.

Reply:
Thanks for your suggestions :)
Actually, it's my mistake and you caught it :D

Since my original intention was the first option you said, I will fix this by removing `toString` method!